build-libs/redirects.js

```diff
@@ -11,9 +11,14 @@ const {
 const fetchGithubFile = require('./fetch-github-file')
 const { isContentDeployPreview } = require('../src/lib/env-checks')
 const buildBetaProductOptInRedirects = require('./build-beta-opt-in-redirect')
+const { loadHashiConfigForEnvironment } = require('../config')
+
+require('isomorphic-unfetch')
```

Comment:
What's the need for this `require`?

Reply:
It makes `fetch` available, which isn't defined by default in node. I was running into some type errors trying to import fetch, so just did this...

Reply:
_cough_ NODE 18 _cough_ 😄

Reply:
![Screen Shot 2022-10-04 at 11 39 46 AM](https://user-images.githubusercontent.com/1289701/193877198-5067eb65-6e80-4eff-85e2-64e76792c12b.png)

🥲

Reply:
>It makes fetch available, which isn't defined by default in node. I was running into some type errors trying to import fetch, so just did this...

The Next.js runtime polyfills `fetch` right?
But `build-libs/redirects.js` runs outside of this runtime, in the standard `node` runtime?

Reply:
Correct 👍